test/e2e/policies.test-suite/balloons/cri-resmgr.cfg

```diff
@@ -25,6 +25,7 @@ policy:
         Namespaces:
           - "three"
         MinCPUs: 3
+        MaxCPUs: unlimited
```

Comment:
I thought we ditched this change(?)

Reply:
I fixed the problems in the ditched version: parsing quotation and unlimited not being the default if `MaxCPUs` or `MaxBalloons` is missing. There were two reasons for advocating the `Limit` instead of the `int` type in the configuration.

First, value 0 needs to be handled as a special case in all comparisons and arithmetics where these two options are involved. Own type forces to pay attention on these operations, thus protecting programmers from stupid errors.

Second, enabling also other than numbers being entered to these fields was supposed to pave the road for small extensions later on, like allowing the number of CPUs/balloon and the number of balloons being limited to the number of CPUs, NUMA nodes, dies or packages on a system, or a combination of these (like `CPUsPerNUMA` or `CPUsPerPackage`).

If you think enabling a user to give a separate `unlimited` value is unwanted (I totally understand this because it adds quite a lot of code only for a single value for now), I could alternatively define simply `typedef Limit int`, and allow only integers in the user interface?

Reply:
> If you think enabling a user to give a separate unlimited value is unwanted (I totally understand this because it adds quite a lot of code only for a single value for now), I could alternatively define simply typedef Limit int, and allow only integers in the user interface?

I think that would preferred as that's also kind of unrelated to the "topic" of this PR.

I'd suggest to add it in a separate PR when we really want to have it. It might be a bit more subtle than it looks. I.e. I think we should probably use the same type for MinCpus and MinBalloon parameters, too. And then, does the available string values depend on the field type....

Reply:
Limits removed.